ouroboros-consensus/src/Ouroboros/Consensus/Protocol/Abstract.hs

```diff
@@ -221,16 +221,12 @@ class ( Show (ChainState    p)
   --
   -- Default chain selection
   --
-  -- The default simply compares length
+  -- The default preference uses the default comparison. The default comparison
+  -- simply uses the block number.
   --
 
-  default preferCandidate :: HasHeader hdr
-                          => NodeConfig p
-                          -> AnchoredFragment hdr      -- ^ Our chain
-                          -> AnchoredFragment hdr      --` ^ Candidate
-                          -> Bool
-  preferCandidate _ ours cand =
-    AF.compareHeadBlockNo cand ours == GT
+  preferCandidate cfg ours cand =
+    compareCandidates cfg ours cand == LT
```

Comment:
> Shouldn't it be `cand > ours` instead of `ours < cand`?

I think this change was a refactor because of the usual property that `y > x` if and only if `x < y`. Am I wrong to assume that that property of `compareCandidates`? (I'm a bit confused by your comment about `EQ` -- `preferCandidate` before and after the change is `False` if `ours` and `cand` compare as `EQ`.)

My motivation for the change was because I misread the original at a glance: I wasn't expecting the arguments of the function to switch positions in the delegation.